elastic/kibana
# Classify ES|QL verification_exception as user error

Resolves: #187268

This PR classifies the verification_errors thrown by the Elasticsearch Query (ES|QL) rules as user error.

## To verify

Create an Elasticsearch Query with ES|QL and use anon-existing field in the query. 
Let the rule run.
`/api/task_manager/metrics?reset=false` should show a user error under `alerting:__es-query"`

```
"alerting:__es-query":{"success":0,"not_timed_out":1,"total":1,"total_errors":1,"user_errors":1,"framework_errors":0}
```
